api/src/test/java/org/openmrs/module/fgh/mpi/snapshotEventProcessorTest.java

```diff
@@ -0,0 +1,202 @@
+package org.openmrs.module.fgh.mpi;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.openmrs.api.AdministrationService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.debezium.DatabaseEvent;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.powermock.reflect.Whitebox;
+
+import javax.net.ssl.KeyManagerFactory;
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.singletonMap;
+import static org.junit.Assert.*;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.when;
+import static org.openmrs.module.debezium.DatabaseOperation.UPDATE;
+import static org.openmrs.module.fgh.mpi.MpiConstants.*;
+import static org.openmrs.module.fgh.mpi.MpiIntegrationProcessor.*;
+import static org.openmrs.module.fgh.mpi.MpiIntegrationProcessor.ID_PLACEHOLDER;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({ Context.class, MpiUtils.class, FhirUtils.class, MpiContext.class, BaseEventProcessor.class, KeyManagerFactory.class })
+public class snapshotEventProcessorTest {
```

Comment:
classes MUST start with uppercase letter

Reply:
Sorry for this. kkkk